src/core/jit/exec_const.rs

```diff
@@ -52,6 +53,16 @@ impl ConstValueExecutor {
                 .with_errors(vec![Positioned::new(Error::Unknown, Pos::default())]);
         };
 
+        // Attempt to plan authentication
+        let Ok(plan) = AuthPlaner::new(&req_ctx.server.auth)
+            .transform(plan)
+            .to_result()
```

Comment:
This is technically not planning. We are updating the plan for each request.

Reply:
I got it moved to the static query planning generation